runtimes/CMakeLists.txt

```diff
@@ -151,9 +151,11 @@ endif()
 # Avoid checking whether the compiler is working.
 set(LLVM_COMPILER_CHECKED ON)
 
+# This can be used to detect whether we're in the runtimes build.
+set(LLVM_RUNTIMES_BUILD ON)
```

Comment:
Do you still need this part of the patch now? I don't think you do?

Reply:
Doesn't hurt to set it earlier, but I can remove it to make it cleaner.